### PR TITLE
Update multi-chain-settlement.en-US.mdx

### PR DESCRIPTION
**Pull request by ren-ryder**
edits :)

### CODE REVIEW

File: pages/docs/tech-highlights/multi-chain-settlement.en-US.mdx [hunk 1]
```diff
@@ -21,7 +21,7 @@ We use Ethereum as an example to explain how the settlement layer works.
 Rooch deploys the settlement smart contract(`State Commitment Chain` & `Bridge`) on Ethereum, which maintains an off-chain state tree, with only the root of the state tree recorded on the chain, through which users migrate state between Layer1 and Rooch.
 
 1. When a user migrates state from Layer1 to Rooch, the state appears in the state tree of Rooch after the transactions in Layer1 are finalized.
-2. When a user migrates state from Rooch to Layer1, the state is first locked and waits for the Layer2 are finalized (Challenge period to pass or zkProver provide valid proof) before being moved out of the state tree for settlement.
+2. When a user migrates state from Rooch to Layer1, the state is first locked and waits for the Layer2 (Challenge period to pass or zkProver provides valid proof) before being moved out of the state tree for settlement.
```
Comment (Mine77) [inline pages/docs/tech-highlights/multi-chain-settlement.en-US.mdx:24]: Here's what happens
1. the state is first locked in Rooch (Layer2)
2. The proposer will commit the state to Ethereum (Layer1)
3. Waiting for the challenge period to pass or for the zkProver to provide valid proof to Ethereum (Layer1)
4. The state can be moved out of the state tree for settlement

Can you rephrase this sentence?

File: pages/docs/tech-highlights/multi-chain-settlement.en-US.mdx [hunk 2]
```diff
@@ -32,11 +32,11 @@ Our solution is mirror the `State Commitment Chain` from Ethereum to other chain
 
 Then, another challenge is how to verify the state proof of Ethereum on other chains?
 
-We depend on the solution of sync Ethereum state root to other chains. This is a challenge that all cross-chain solutions face. There are some solutions to this problem, such as Ethereum light nodes, Oracles, etc. We can explore this together.
+We depend on the solution of syncing Ethereum state root to other chains. This is a challenge that all cross-chain solutions face. There are some solutions to this problem, such as Ethereum light nodes, Oracles, etc. We can explore this together.
```
Comment (Mine77) [inline pages/docs/tech-highlights/multi-chain-settlement.en-US.mdx:35]: "We can explore this together"
->
"We are working with the community to explore this further"
or something similar

**Review by Mine77 — CHANGES_REQUESTED** [review]

File: pages/docs/tech-highlights/multi-chain-settlement.en-US.mdx (hunk 2 above)
Comment (ren-ryder) [reply pages/docs/tech-highlights/multi-chain-settlement.en-US.mdx:35]: As the project reaches maturity and grows from and alongside community contributions, we hope to continually iterate towards the optimal solution of a multi-chain layer2. 

*** thoughts on this as a closer? The dovetail at the end with multi-chain layer2 may be better represented by cross-chain with or without the layer2 suffix. I do think its a bonus to clarify the community aspect and highlight the involvement of our community in the project

Comment (Mine77) [reply pages/docs/tech-highlights/multi-chain-settlement.en-US.mdx:35]: Sounds good to me. Can you append another commit to this pr?
